Documentation/developers/codegen.rst

```diff
@@ -0,0 +1,62 @@
+Polycube Automatic Code Generation Tools
+========================================
+
+Generating a service stub
```

Comment:
Does it make sense adding an explicit pointer to `polycube-codegen` repository?
A Docker is provided to the user, but no pointer to the repository, in case he wants to install codegen tools locally.

Reply:
Yes, it does!